bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/TpeImageDisplayToolBar.java

```diff
@@ -96,7 +77,6 @@ protected JButton makeImageButton() {
     public void update() {
         super.update();
         makeManualGuideStarButton();
-        makeAutoGuideStarButton();
```

Comment:
Not any more: they have all been removed.
I'm not sure if this could cause problems if the BAGS code is working properly, but it may be worthwhile to consider a (nonobvious - used only in case of serious issues) mechanism to force a lookup.